FWCore/ParameterSet/python/Config.py

```diff
@@ -1769,17 +1790,17 @@ def testSubProcess(self):
 
 childProcess = process
 process = parentProcess
-process.subProcess = cms.SubProcess( process = childProcess, SelectEvents = cms.untracked.PSet(
+process.addSubProcess(cms.SubProcess(process = childProcess, SelectEvents = cms.untracked.PSet(
 
-), outputCommands = cms.untracked.vstring())
+), outputCommands = cms.untracked.vstring()))
 """
-            equalD = equalD.replace("parentProcess","parentProcess"+str(hash(process.subProcess)))
-            self.assertEqual(d,equalD)
-            p = TestMakePSet()
-            process.subProcess.insertInto(p,"dummy")
-            self.assertEqual((True,['a']),p.values["@sub_process"][1].values["process"][1].values['@all_modules'])
-            self.assertEqual((True,['p']),p.values["@sub_process"][1].values["process"][1].values['@paths'])
-            self.assertEqual({'@service_type':(True,'Foo')}, p.values["@sub_process"][1].values["process"][1].values["services"][1][0].values)
+#           equalD = equalD.replace("parentProcess","parentProcess"+str(hash(process.subProcesses_[0])))
```

Comment:
Because that test failed at the time, and I forgot to fix it. The test is now fixed, and is no longer commented out un the updated PR.